### PR TITLE
chore: adjust 'packages/mail/src/mail.d.ts'

### PR DESCRIPTION
**Pull request by collierrgbsitisfise**
Error for callBack function of 'send' and 'sendMultiple' methods is not
described as nullable, but it should cause error is null when emails
where successfully delivered

### CODE REVIEW

File: packages/mail/src/mail.d.ts [hunk 1]
```diff
@@ -32,12 +32,12 @@ declare class MailService {
...
    * Send email
    */
-  send(data: MailDataRequired | MailDataRequired[], isMultiple?: boolean, cb?: (err: Error | ResponseError, result: [ClientResponse, {}]) => void): Promise<[ClientResponse, {}]>;
+  send(data: MailDataRequired | MailDataRequired[], isMultiple?: boolean, cb?: (err: Error | ResponseError | null, result: [ClientResponse, {}]) => void): Promise<[ClientResponse, {}]>;
 
   /**
    * Send multiple emails (shortcut)
    */
-  sendMultiple(data: MailDataRequired, cb?: (error: Error | ResponseError, result: [ClientResponse, {}]) => void): Promise<[ClientResponse, {}]>;
+  sendMultiple(data: MailDataRequired, cb?: (error: Error | ResponseError | null, result: [ClientResponse, {}]) => void): Promise<[ClientResponse, {}]>;
```
Comment (JenniferMah) [inline packages/mail/src/mail.d.ts:40]: Hi @collierrgbsitisfise! Thanks for the contribution. Can you add some additional tests to verify that error is null when emails are successfully delivered?

**Review by JenniferMah — DISMISSED** [review]

File: packages/mail/src/mail.d.ts (hunk 1 above)
Comment (collierrgbsitisfise) [reply packages/mail/src/mail.d.ts:40]: @JenniferMah done, add more test cases for mail functionality

**Review by JenniferMah — APPROVED** [review]
Thanks for adding tests @collierrgbsitisfise!
